installer_scripts/install-yb-voyager

```diff
@@ -173,6 +183,36 @@ macos_main() {
 # COMMON
 #=============================================================================
 
+check_java() {
+		if [ -z "$JAVA_HOME" ]; then
+			JAVA_BINARY="java"
+		else
+			JAVA_BINARY="$JAVA_HOME/bin/java"
+		fi
+
+        MIN_REQUIRED_MAJOR_VERSION='11'
+		JAVA_MAJOR_VER=$(${JAVA_BINARY} -version 2>&1 | awk -F '"' '/version/ {print $2}' | awk -F. '{print $1}')
+		
+        if ([ -n "$JAVA_MAJOR_VER" ] && (( 10#${JAVA_MAJOR_VER} >= 10#${MIN_REQUIRED_MAJOR_VERSION} )) ) #integer compare of versions.
+        then
+        	output "Found sufficient java version = ${JAVA_MAJOR_VER}"
+        else
+        	output "ERROR: Java not found or insuffiencient version ${JAVA_MAJOR_VER}. Please install java>=${MIN_REQUIRED_MAJOR_VERSION}"
+			exit 1;
+        fi
+}
+
+install_debezium_server() {
+	output "Installing debezium"
+	debezium_server_filename="debezium-server-${DEBEZIUM_TARBALL_VERSION}.tar.gz"
+	# download
+	wget -nv "https://github.com/yugabyte/debezium/releases/download/voyager-debezium/${debezium_server_filename}"
```

Comment:
As discussed in https://yugabyte.slack.com/archives/C04MC0N7B1A/p1684320717861679?thread_ts=1684315471.715119&cid=C04MC0N7B1A, make this `wget -nv "https://github.com/yugabyte/debezium/releases/download/voyager-debezium-${DEBEZIUM_TARBALL_VERSION}/${debezium_server_filename}"`

Reply:
Done